src/types/io.ts

```diff
@@ -191,11 +191,16 @@ export type AoGatewayServices =
 export type AoGatewayDelegates = Record<WalletAddress, AoGatewayDelegate>;
 export type AoGatewayDelegateAllowList = WalletAddress[];
 
+export type AoWalletVault = AoVaultData & {
+  address: WalletAddress;
+  vaultId: string;
+};
+
 export type AoGateway = {
   settings: AoGatewaySettings;
   stats: AoGatewayStats;
   totalDelegatedStake: number;
-  vaults: Record<WalletAddress, AoVaultData>;
+  vaults: AoWalletVault;
```

Comment:
this is is till the previous types. AoWalletVault is only returned by getVault/getVaults API atm

Reply:
3f281ed

Reply:
vaultId is not be represented in that API?